app/services/people/finder.rb

```diff
@@ -15,25 +15,15 @@ def call
   private
 
     def apply_filters(scope)
-      scope = scope.joins(:profiles)
-
       if filter_params.key?(:police_national_computer)
-        scope = scope.where('profiles.profile_identifiers @> ?', police_national_computer.to_json)
+        scope = scope.where(police_national_computer: filter_params[:police_national_computer])
```

Comment:
note that the previous search was a _range_ search (`@>`) whereas the new search is an exact match. Is this intended?

Reply:
Yeah, the values are totally unique and a person only ever has one unique identifier for each type.

Reply:
what if the front end wants to search with a partial identifier, e.g. `AB1` rather than `AB12345678` etc?